QSPIFBlockDevice.cpp

```diff
@@ -827,79 +826,76 @@ int QSPIFBlockDevice::_sfdp_set_quad_enabled(uint8_t *basic_param_table_ptr)
 
     switch (qer_value) {
         case 0:
-            tr_debug("DEBUG: Device Does not Have a QE Bit, continue based on Read Inst");
+            tr_debug("Device Does not Have a QE Bit, continue based on Read Inst");
             return 0;
 
         case 1:
         case 4:
-            status_reg_setup[0] = 0;
-            status_reg_setup[1] = 0x02;  //Bit 1 of Status Reg 2
-            tr_debug("DEBUG: Setting QE Bit, Bit 1 of Status Reg 2");
+            status_reg_setup[1] = (0x1 << 1);  //Bit 1 of Status Reg 2
```

Comment:
Why do you need the unnecessary shifting? I intentionally wrote e.g. 0x40 and not (0x1 << 6) to avoid unnecessary  shifting commands.

Reply:
Shifting won’t add too much processing, but helps in updating the bits with less chance of mistakes. Did this as part of verification, can revert back if you strongly want to avoid

Reply:
I'd prefer it to remain as was, if possible - thanks.

Reply:
Sure, will update it

Reply:
Done